.github/workflows/audit.yaml

```diff
@@ -0,0 +1,60 @@
+name: audit
+
+on: [pull_request] 
+
+jobs:
+  audit:
+    name: Audit packages
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: oke-py/npm-audit-action@v2
+        with:
```

Comment:
I noticed audit was failing on this branch. Also the same concerns here; should it be a separate workflow?